indico/htdocs/js/utils/forms.js

```diff
@@ -77,9 +102,34 @@
             });
         }).on('change input', function() {
             var $this = $(this);
-            var untouched = $this.serialize() == $this.data('initialData');
+            var untouched = $this.serialize() === $this.data('initialData');
+            var $cornerMessage = $('.corner-message');
             $this.find('[data-disabled-until-change]').prop('disabled', untouched);
             $this.closest('form').data('fieldsChanged', !untouched);
+            if ($this.find('[data-save-reminder]').length) {
+                if (!isElementInView($this.find('[data-save-reminder]'))
+                && !untouched && !$cornerMessage.length) {
```

Comment:
Small code style improvement: If you want to break the `if` statement in 2 lines, align the second line like this:
``` js
if (!isElementInView($this.find('[data-save-reminder]'))
    && !untouched && !$cornerMessage.length) {
    ....
}
```